models/booking.js

```diff
@@ -3,20 +3,55 @@ const daoBooking = require("../daos/booking");
 module.exports = {
   getAllByUserId,
   getAllByRestaurantId,
+  filterAllByRestaurantId,
   getOneById,
   createBooking,
   updateBooking,
   deleteBooking,
 };
 
 function getAllByUserId(id) {
-  // return daoBooking.find({ user: id });
-  return daoBooking.find({});
+  return daoBooking
+    .find({
+      dateTime: {
+        $gte: new Date(),
```

Comment:
wasn't able to test this function. will KIV for ltr when user data is connected!

PS - noticed that this segment uses lodash.. went to read up further. feels like a good library for utility functions. thank you for this new info!